scripts/run-bench.sh

```diff
@@ -94,6 +93,7 @@ hey -n $numReqs -c $numParallel -m POST \
     | tee -a BENCHMARKS.md
 
 
+# shellcheck disable=SC2006
```

Comment:
I set it to disable because it is a backtick that is written in markdown.

https://github.com/koalaman/shellcheck/wiki/SC2006